src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -240,7 +240,7 @@
     <value>A CodeFixProvider that intends to support fix all occurrences must classify the registered code actions into equivalence classes by assigning it an explicit, non-null equivalence key which is unique across all registered code actions by this fixer. This enables the FixAllProvider to fix all diagnostics in the required scope by applying code actions from this fixer that are in the equivalence class of the trigger code action.</value>
```

Comment:
Can you please also update this description resource similarly?

Reply:
Done.
I didn't fix this, but we might also want to reword this one since it says the value must be non-null, which as we've discussed in https://github.com/dotnet/roslyn/pull/27989 it actually needn't be.

Reply:
I think we still want the analyzer diagnostic to recommend using a non-null equivalence key for clarity and maintenance. It will work for null value, but this seems cleaner. If this produces too much of noise, and we get feedback we can do as per your suggestion.